src/Nethermind/Nethermind.Serialization.Rlp/RlpStream.cs

```diff
@@ -52,6 +52,11 @@ public RlpStream(in CappedArray<byte> data)
             _data = data;
         }
 
+        public RlpStream Clone()
+        {
+            return new(_data);
```

Comment:
Can cloned RlpStreams be structs? (Didn't want to disappoint).

Reply:
Follow up for the perf https://github.com/NethermindEth/nethermind/pull/6584